### ISSUE TITLE
DMA usage ?

### ISSUE THREAD

**Issue by toncho11** [aster94/logic-analyzer#14]
Hi,

A project like this should use DMA (Direct Access Memory) in order to lower the delays in acquiring the data? [Example Ardiono Zero Read with DMA](https://github.com/manitou48/ZERO/blob/master/adcdma.ino
)
The STM32F1 you refer has a DMA controller indeed. [STM32F0/F1/F3/G0/Lx Series DMA controller](https://www.st.com/resource/en/application_note/cd00160362-using-the-stm32f0f1f3g0lx-series-dma-controller-stmicroelectronics.pdf) But you might not be able to use the DMA from the Arduino IDE ? You might need to use the ST tools to utilize it ?

**Comment by aster94** [comment]
Yes, a DMA would be better, on stm32 it is usable also in the arduino IDE, on notepad or on any other editor, but I don't plan to add it
